src/main/keybinds.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import child_process from "node:child_process";
+import fs from "node:fs";
+import path from "node:path";
+
+import net from "net";
+import { IpcEvents } from "shared/IpcEvents";
+
+import { mainWin } from "./mainWindow";
+
+const xdgRuntimeDir = process.env.XDG_RUNTIME_DIR || process.env.TMP || "/tmp";
+const socketFile = path.join(xdgRuntimeDir, "vesktop-ipc");
+
+export function initKeybinds() {
+    child_process.spawnSync("mkfifo", [socketFile]);
```

Comment:
this could potentially throw for various reasons, as could fs.open

please wrap it in try catch and add reasonable logging

you can try the error handling by passing bad paths like `/pipe`